crud.py

```diff
@@ -9,19 +9,6 @@
 # from loguru import logger
 
 
-async def check_lnaddress_update(username: str, id: str) -> bool:
-    # check no duplicates for lnaddress when updating an username
-    row = await db.fetchall(
-        "SELECT username FROM lnurlp.pay_links WHERE username = ? AND id = ?",
-        (username, id),
-    )
-    if len(row) > 1:
-        assert False, "Username already exists. Try a different one."
-        return
-    else:
-        return True
-
-
```

Comment:
`check_lnaddress_update(username: str, id: str)` doesn't make sense since `id`/`link_id` is not changed during the update, and the couple `username` and `id` is expected to be *always* unique.